### ISSUE TITLE
Warning when instantiating Plot with no arguments

### ISSUE THREAD

**Issue by birdsarah** [bokeh/bokeh#2441]
The following code:

````
from bokeh.io import output_file, show
from bokeh.models import Plot
output_file('test.html')
p = Plot()
show(p)
````
Will give you the dreaded white screen of death. 

If you use output_notebook you fair slightly better and get:

````
Javascript error adding output!
TypeError: y is null
See your browser Javascript console for more details.
````

Should we have a warning about instantiating empty plot models? Or should we just rely on whatever the solution is for #2392.

**Comment by damianavila** [comment]
As I said in #2392, there are probably a lot of causes for the "screen of death"... not sure about which is the best general approach here... but it seems that we need to collect "detectors" for each of these problems...

**Comment by birdsarah** [comment]
closing this as now get warnings from the new validation in 0.9.1 like

```python
ERROR:/opt/miniconda/envs/ipython_bokeh_34/lib/python3.4/site-packages/bokeh/validation/check.py:W-1000 (MISSING_RENDERERS): Plot has no renderers: Plot, ViewModel:Plot, ref _id: 0c5f3dbb-c65a-42bb-a9a9-0ac91c9ffcf3
ERROR:/opt/miniconda/envs/ipython_bokeh_34/lib/python3.4/site-packages/bokeh/validation/check.py:W-1001 (NO_GLYPH_RENDERERS): Plot has no glyph renderers: Plot, ViewModel:Plot, ref _id: 0c5f3dbb-c65a-42bb-a9a9-0ac91c9ffcf3

````

**Comment by birdsarah** [comment]
closed by #2470
